g11n-ws/vip-common/src/main/java/com/vmware/vip/common/constants/ConstantsMsg.java

```diff
@@ -10,6 +10,7 @@
  */
 public class ConstantsMsg {
    // message
+   public static final String TRANS_GET_FAILD = "Faild to get translation from data for %s";
```

Comment:
change it to "Failed to get translations from the bundles for %s".

Reply:
done